dump/dump_watch.cpp

```diff
@@ -42,6 +42,15 @@ void DumpWatch::interfaceAdded(sdbusplus::message::message& msg)
         sdbusplus::message::object_path objPath;
         DBusInteracesMap interfaces;
         msg.read(objPath, interfaces);
+        DBusInteracesMap::iterator iter = interfaces.begin();
+        if (interfaces.find("com.ibm.Dump.Entry.Hostboot") ==
+                interfaces.end() &&
+            interfaces.find("com.ibm.Dump.Entry.Hardware") ==
+                interfaces.end() &&
+            interfaces.find("com.ibm.Dump.Entry.SBE") == interfaces.end() &&
+            interfaces.find("xyz.openbmc_project.Dump.Entry.BMC") ==
+                interfaces.end())
+            return;
```

Comment:
watch need to be modified stating it want to watch an object which has both system && hardware, system && hostboot and e.t.c.